src/demo-app/demo-app-module.ts

```diff
@@ -37,6 +37,7 @@ import {TabsDemo} from './tabs/tab-group-demo';
   imports: [
     BrowserModule,
     FormsModule,
+    ReactiveFormsModule,
```

Comment:
Given that in the demo, you ended up with `ngModel`, you can remove `ReactiveFormsModule` from the demo module.